decidim-core/app/jobs/decidim/machine_translation_create_fields_job.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Decidim
+  class MachineTranslationCreateFieldsJob < ApplicationJob
+    queue_as :default
+
+    def perform(id, resource_type, field_name, field_value, locale)
+      Decidim::TranslatedField.create!(
```

Comment:
If we pass the object instead of the class name and id we can set it as `translated_resource: resource`